cli.py

```diff
@@ -0,0 +1,78 @@
+"""TrustyMail A tool for scanning DNS mail records for evaluating security.
+Usage:
+  trustymail (INPUT ...) [options]
+  trustymail (INPUT ...) [--output=OUTFILE] [--timeout=TIMEOUT] [--mx] [--spf] [--dmarc] [--debug]
+  trustymail (-h | --help)
+Options:
```

Comment:
Can you cover installation of dependancies (via a requirements.txt)? 

We'll post to PyPI once we have a 0.1 and can then update this.